lib/devices/shellyplus1.js

```diff
@@ -0,0 +1,123 @@
+/* jshint -W097 */
+/* jshint -W030 */
+/* jshint strict:true */
+/* jslint node: true */
+/* jslint esversion: 6 */
+'use strict';
+
+const shellyHelper = require('../shelly-helper');
+
+/**
+ * Shelly Plus 1 / shellyplus1
+ */
+let shellyplus1 = {
+  'Relay0.Switch': {
+    mqtt: {
+      mqtt_publish: '<mqttprefix>/events/rpc',
+      mqtt_publish_funct: (value) => {
+        const valueObj = JSON.parse(value);
+        if (
+          Object.prototype.hasOwnProperty.call(valueObj, 'method') &&
```

Comment:
I know why the code is that way, but with JSON.parse I do not really see the risk and it makes the code that bad readable ... but here just "comment" ... all other places are mot written that way

Reply:
I don't like that, too. But every payload looks nearly identical. If you have any idea how to beautify that part - let me know

Reply:
I was more talking about the Object.prototype.hasOwnProperty.call stuff ... valueObj.method ? :-)